ls1intum/Artemis
# Fix Sorting in Student Exams

Fixes #2886 




### Checklist

### Motivation and Context


In the student exams view, it was not possible to filter for submissionDate correctly anymore.
When clicking the sort button next to the submission date, the dates stayed the same while other column sorted accordingly. Therefore, student exams seem to have different submissionDates as they actually have

### Description

The date has to be manually formatted in the component.ts, since the artemisDate pipe seems to not be able to apply sorting changes. 

### Steps for Testing


1. Log in to Artemis
2. Move into a exam with student exams (with multiple and different submissionDates)
3. Keep in mind which student exam has which submissionDate
3. Try to sort after submissionDate and check that the sorting works (so that each student exam has its correct submissionDate)
